openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -413,8 +413,8 @@ There are two possible ways to request issuance of a specific Credential type in
 
 The request parameter `authorization_details` defined in Section 2 of [@!RFC9396] MUST be used to convey the details about the Credentials the Wallet wants to obtain. This specification introduces a new authorization details type `openid_credential` and defines the following parameters to be used with this authorization details type:
 
-* `type` REQUIRED. String that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
-* `format`: REQUIRED. String representing the format in which the Credential is requested to be issued. This Credential format identifier determines further claims in the authorization details object specifically used to identify the Credential type to be issued. This specification defines Credential Format Profiles in (#format_profiles).
+* `type`: REQUIRED. String that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
+* `credential`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credentials_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credentials_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

Comment:
```suggestion
* `credential_identifier`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credentials_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credentials_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

Reply:
Calling something `credential` and it is not a credential strikes me.

Reply:
>  Calling something `credential` and it is not a credential strikes me.

Concur with the sentiment. But `credential_identifier` is (or will be) used elsewhere, no?

Reply:
The problem is that we don't have a clear concept here and therefore no clear name. We discussed this on the last call and agreed that we need to fix this in Issue #91.

Reply:
I named it `credential_description` beforehand, argument was that credential_offer references the same thing and it's named `credentials` there. `credential_identifier` is used in Token Response/Credential Request.

Reply:
if we can agree on something like `credential_configuration`, I am ok. but it cannot be `credential_identifier`.

Reply:
discussed on DCP WG call, agreed on `credentials_supported_id` for now. might change if `credentials_supported` parameter itself gets renamed...